Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -92,6 +93,11 @@ public static class Loader {
 
             public static bool AutoLoadNewMods { get; internal set; }
 
+            /// <summary>
+            /// Used by <see cref="OuiModTogglerProgress"/>
+            /// </summary>
+            internal static List<string> NewlyUnblacklistedMods;
```

Comment:
this is here because I essentially need to pass data from one oui to another (the list of mods to enable from `OuiModToggler` to `OuiModTogglerProgress`). i don't like this solution but i couldn't see a better option than just a static list somewhere

Reply:
other option i considered was keeping it as members of `OuiModToggler` and just doing a cast check for `from` in `OuiModTogglerProgress.Enter` (while also not erasing the data in `OuiModtoggler.Leave`) but that seems equally bad

Reply:
oh NVM had realization